config/systolic.mk

```diff
@@ -29,6 +29,6 @@ seq_mem_size ?= 2048
 ##  Xqueues configuration  ##
 #############################
 
-# XQueue extension's queue size in each memory bank (in words)
-# (assume banking factor of 4)
+# XQueue extension's queue size (in queue entries; an entry can contain
+# multiple words) in each memory bank (assume banking factor of 4)
```

Comment:
Will this still be true with the hardware extension?

Reply:
Right, not really, should be just implemented in software